src/forcegraph-kapsule.js

```diff
@@ -1026,7 +1034,14 @@ export default Kapsule({
         layout.graph = graph; // Attach graph reference to layout
       }
 
-      for (let i = 0; i < state.warmupTicks; i++) { layout[isD3Sim ? 'tick' : 'step'](); } // Initial ticks before starting to render
+      for (
+        let i = 0;
+        i < state.warmupTicks &&
+        !(isD3Sim && (state.d3ForceLayout.alpha() < state.d3AlphaMin));
```

Comment:
For not affecting backwards performance, I'd suggest to check if `d3AlphaMin > 0` before comparing it with the d3 alpha.

Reply:
Do you mean before the hole block
```
!(isD3Sim && (state.d3ForceLayout.alpha() < state.d3AlphaMin))
```

or just the second statement?

```
(state.d3ForceLayout.alpha() < state.d3AlphaMin)
```


And I guess that it should also be checked in the layoutTick, right?

Reply:
I'd do 
```js
!(isD3Sim && state.d3AlphaMin > 0 && state.d3ForceLayout.alpha() < state.d3AlphaMin)
```

And yes, it should be the same check on the layoutTick.